module-4/backend/week_two.md

```diff
@@ -5,18 +5,52 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+  - String interpolation
+  - Class
+    - Contstructor setup  
+2. What's the difference between asynchronous and synchronous JavaScript?
+  - Synchronous evaluates line - line in a linear way
+  - Asynchronous doesn't do that ^^
 3. What are the four pillars of Object Oriented programming?
+  - Abstraction
+  - Encapsulation
+  - Inheritance
+  - Polymorphism 
 4. What are some tools available in JavaScript to help you write object oriented code?
+  - Constructor functions
+  - JS {} objects
+  - A good attitude
 5. What are some key concepts to be aware of when refactoring your JavaScript?
+  - What "this" is at any given time
 6. What's a callback function and what are some reasons when we use/need callback functions?
+  - A callback function is a function that isn't invoked unless a different function has been invoked.
 7. What's the scope of variables in Javascript?
+  - Global || Local
 8. What's the difference between `==` and `===` in JavaScript?
+  - The `===` compares the exact values and `==` is like one level difference.
```

Comment:
what is that one level difference? what's each operator actually using to check equality?